synctl

```diff
@@ -2359,171 +2366,154 @@ class UpdateSyntheticTest(SyntheticTest):
             payload = json_file.read()
             return payload
 
-    def update_label(self, payload, label):
+    def update_label(self, label):
         """label"""
         if label is None:
             print("no label")
         else:
-            payload["label"] = label
-            return payload
-    def update_description(self, payload, description):
+            self.update_test_config["label"] = label
+
+    def update_description(self, description):
         """description"""
         if description is None:
             print("no description")
         else:
-            payload["description"] = description
-            return payload
+            self.update_test_config["description"] = description
 
-    def update_record_video(self, payload, record_video=False):
+    def update_record_video(self, record_video=False):
         if record_video is not None and record_video.upper() in ("TRUE", "FALSE"):
-            payload["configuration"]["recordVideo"] = record_video
-            return payload
+            self.update_test_config["configuration"]["recordVideo"] = record_video
 
-    def update_active(self, payload, active):
+    def update_active(self, active):
         """active"""
         if active is None:
-            return
+            print("active should not be none")
         if active == "false":
-            payload["active"] = False
+            self.update_test_config["active"] = False
         if active == "true":
-            payload["active"] = True
-        return payload
+            self.update_test_config["active"] = True
 
-    def update_frequency(self, payload, frequency):
-        """patch frequency, 1, 120"""
+    def update_frequency(self, frequency):
+        """update frequency, 1, 120"""
         if frequency is None:
             frequency = 15
         if frequency > 0 and frequency <= 120:
-            payload["testFrequency"] = frequency
-            return payload
+            self.update_test_config["testFrequency"] = frequency
```

Comment:
Use if else if here, if frequency is None, just show a wanning message, otherwise it will update frequency to 15. For example, if frequency is None, and frequency of test configuration is 1, it will change to 15 which is not expected.